src/widgets/myspeed/component.jsx

```diff
@@ -0,0 +1,64 @@
+import { useTranslation } from "next-i18next";
+
+import Container from "components/services/widget/container";
+import Block from "components/services/widget/block";
+import useWidgetAPI from "utils/proxy/use-widget-api";
+
+export default function Component({ service }) {
+  const { t } = useTranslation();
+  const { widget } = service;
+  const { data, error } = useWidgetAPI(widget, "info");
+
+  if (error) {
+    return <Container service={service} error={error} />;
+  }
+
+  if (!data) {
```

Comment:
```suggestion
  if (!data || (data && data.length === 0)) {
```

If no tests. I suppose it could be handled as an error too but this seems simpler. Without anything tho you'll get:
<img width="973" alt="Screenshot 2024-06-23 at 9 47 46 AM" src="https://github.com/gethomepage/homepage/assets/4887959/93a34c84-ae8e-4457-9f9f-4a44837a1bfa">